### PR TITLE
quic blog

### PR DESCRIPTION
**Pull request by stephamon**
In a comment on the pull request, include the following information and check off the list.

 - [ ] Mini bio
  - Soft capped to 75 words
 - [ ] Desired date of publishing
  - Make sure this matches the `date` in the front matter
 - [ ] Social media contact info
 - [ ] Redaction: (yes|no)
  - Allow redaction, editing or structure changes by technical writer staff
 - [ ] Comments: (yes|no)
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

**Comment by stephamon** [comment]
build me

**Comment by rackernexus** [comment]
Build 1278 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-2e78bb3c5/). valid for 14 days

**Comment by stephamon** [comment]
Hey @catlook this should be ready to go if you approve :)

**Comment by catlook** [comment]
build me

**Comment by rackernexus** [comment]
Build 1281 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-be17c27e5/). valid for 14 days

**Comment by catlook** [comment]
build me

**Comment by catlook** [comment]
build me

**Comment by rackernexus** [comment]
Build 1283 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-db7320af0/). valid for 14 days

File: _posts/2020-06-24-quic-a-game-changer.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+### Where does QUIC fit?
+
+Let’s understand now where the QUIC protocol fits in the Open Systems Interconnection (OSI) layer. 
+QUIC is a cross-layer protocol that rides on top of UDP. It becomes its own protocol by stealing of
+all the good things from TCP (congestion control and loss recovery), TLS (cryptography and handshake),
+and all of the control aspects of HTTP/2 (multi-streaming).
+
+![QUIC protocol]({% asset_path 2020-06-24-quick-a-game-changer/quic-protocol.png %})
+
+### Benefits
```
Comment (mustapha-benmbarek) [inline _posts/2020-06-24-quic-a-game-changer.md:81]: What are the benefits of QUIC ?

File: _posts/2020-06-24-quic-a-game-changer.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+- **Reliability (Congestion control)**: QUIC is reliable just like TCP, and it
+incorporates all the best practices, building on years of learning with
+TCP. Unlike TCP, the QUIC protocol does not rely on a specific congestion
+control algorithm, and its implementation has a pluggable interface to allow
+experimentation and flexibility. Endpoints can unilaterally choose a different
+algorithm to use, such as CUBIC. It also improves loss recovery by using
+unique packet numbers to avoid retransmission ambiguity and by using explicit
+signaling in acknowledgements (ACKs) for accurate RTT measurements.
+
+### Challenges
```
Comment (mustapha-benmbarek) [inline _posts/2020-06-24-quic-a-game-changer.md:124]: What are the challenges ?

**Comment by catlook** [comment]
build me

**Comment by rackernexus** [comment]
Build 1289 was successful. Staging content [Preview!](https://developer-revs.staging.deconst.org/build-0e7509568/). valid for 14 days

**Pull request merged**